service/raft/raft_group0.cc

```diff
@@ -671,6 +671,30 @@ future<> raft_group0::finish_setup_after_join() {
     });
 }
 
+future<> raft_group0::become_nonvoter() {
```

Comment:
Typo: "If the procedure fails before after step..."

Reply:
Why `become_nonvoter()` does not call `make_nonvoter() ` with itself?

Reply:
The only practical difference between these functions is the logging events. `become_nonvoter` talks about "me" (this node), `make_nonvoter` talks about "them" (a different node). The common part which does the actual config change is already extracted to a separate private function, `make_raft_config_nonvoter`.